PhysicsTools/IsolationAlgos/plugins/CITKPFIsolationSumProducerForPUPPI.cc

```diff
@@ -76,7 +76,10 @@ namespace citk {
         puppiValueMapToken_ = mayConsume<edm::ValueMap<float>>(c.getParameter<edm::InputTag>("puppiValueMap")); //getting token for puppiValueMap
         useValueMapForPUPPI = true;
       }
-      else useValueMapForPUPPI = false;
+      else {
+	useValueMapForPUPPI = false;
+	usePUPPINoLepton = (c.exists("usePUPPINoLepton") ? c.getParameter<bool>("usePUPPINoLepton") : false);
```

Comment:
also, this appears to be a bugfix to the ``` usePUPPINoLepton ```, which should explain why the electron puppi isolations have changed.